pages/Documents/ConversationalAI/Third-PartyBots/CustomEndpoint/basic-content.md

```diff
@@ -0,0 +1,506 @@
+
+{: .important}
+If Images are sent in Rich content, then their URLs must be added to a whitelist via internal
+LivePerson configuration (Houston: `messaging.rich.content.valid.urls`). Please note that you
+must add all possible domains to this list manually as wildcards are not supported. Moreover,
+All domains must be HTTPS secure.
+
+An example of a Rich Content message response along with any optional metadata and analytical
+information (intents) can be seen in Figure 2.5 below. More information on responses expected
+by Third-Party Bot Connector can be found at [API Service Specification](https://github.com/LivePersonInc/third-party-bots-custom-endpoint-reference-service)
+
+{: .important}
+The response is expected to be sent by the [Get Conversation Events](third-party-bots-custom-endpoint-basic-content.html#custom-endpoint-ce-service-methods)
+method of the Custom Endpoint service please ensure valid structured content is sent by using our
+[Conversation Tester feature](third-party-bots-conversation-tester.html) or using
+[this tool](https://livepersoninc.github.io/json-pollock/editor/).
+
+```json
+{
+  "response": [
+    {
+      "type": "STRUCTURED_CONTENT",
+      "data": {
+        "metadata": [{ "type": "ExternalId", "id": "ABCD1234" }],
+        "structuredContent": {
+          "type": "vertical",
+          "elements": [
+            {
+              "type": "button",
+              "click": {
+                "actions": [
+                  { "text": "Recommend me a movie", "type": "publishText" }
+                ]
+              },
+              "title": "Recommend a movie"
+            }
+          ]
+        }
+      }
+    }
+  ],
+  "analytics": {
+    "intents": [
+      {
+        "id": "base-structured-content",
+        "description": "Structured Content Message",
+        "confidenceScore": 1
+      }
+    ]
+  }
+}
+```
+
+Figure 2.5 Showing an example response of Structure Content Message
+
+### Sending Quick Replies (Structured Content)
+
+{: .important}
+**Please note** Quick Replies are only supported in Messaging Conversations.
+
+Quick Replies is a special type of Structured Content. It is a message sent along with predefined answers.
+For detailed information on Quick Replies check out the documentation for the specific channel ([Mobile SDK and Web](mobile-sdk-and-web-templates-quick-replies-template.html), [Facebook Messenger](facebook-messenger-templates-quick-replies-template.html),
+[Google RCS Business Messaging](google-rcs-business-messaging-templates-quick-replies-template.html)).
+
+An example of a Quick Replies message response along with any optional metadata and analytical
+information (intents) can be seen below in Figure 2.6. More information on responses expected
+by Third-Party Bot Connector can be found at [API Service Specification](https://github.com/LivePersonInc/third-party-bots-custom-endpoint-reference-service)
```

Comment:
Doe we really nee to mention this on every content description?

Reply:
I think yes if someone came only for the Custom Endpoint documentation then it should be there

Reply:
It's not just on the Custom Endpoint once, it's on each content of the custom endpoint

Reply:
Sorry I mean if someone comes to only a specific content description. I will keep it like that